tests/test_approve.py

```diff
@@ -193,7 +193,7 @@ def test_single_incident(fake_qem, caplog):
         ],
     )
     approver(incident=1)
-    assert len(caplog.records) == 5
+    assert len(caplog.records) >= 1, "we rely on log messages"
```

Comment:
I suggest to also add short docstring to document which behavior this test is testing

Reply:
Yes but I don't think this comment helps here.

Reply:
The following asserts check that the expected messages are seen. Arguably this check is completely redundant.

Reply:
Well, the check for the length of the arrays checks helps to prevent confusing IndexError exceptions